src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
agreed with @grantmcconnaughey, an extra query will be fired off, even using `values_list()` the queryset is still lazily evaluated. If you want to use `primary_keys` you would need to force evaluation, e.g.
```
primary_keys = list(queryset.values_list(queryset.model._meta.pk.name, flat=True))
```

then `primary_keys[0]` is a constant time call, but the issue here is that if for some reason the primary key isn't in the first slot we would get unexpected behavior, whereas `queryset.model._meta.pk` is explicitly getting the primary key defined for that model through the `Meta` options

Maybe the solution is updated the previous